doc/md_src_files/060_decision-trees.md

```diff
@@ -23,31 +23,31 @@ How this decision information might be stored or communicated is the topic of su
 
 ## Supplier Tree
 
-The example supplier tree [PDF](https://github.com/CERTCC/SSVC/blob/main/doc/graphics/ssvc_2_supplier.pdf) shows the proposed prioritization decision tree for the supplier. Both supplier and deployer trees use the above decision point definitions. Each tree is a compact way of expressing assertions or hypotheses about the relative priority of different situations. Each tree organizes how we propose a stakeholder should treat these situations. Rectangles are decision points, and triangles represent outcomes. The values for each decision point are different, as described above. Outcomes are priority decisions (defer, scheduled, out-of-cycle, immediate); outcome triangles are color coded:
+The example supplier tree [PDF](../graphics/ssvc_2_supplier.pdf) shows the proposed prioritization decision tree for the supplier. Both supplier and deployer trees use the above decision point definitions. Each tree is a compact way of expressing assertions or hypotheses about the relative priority of different situations. Each tree organizes how we propose a stakeholder should treat these situations. Rectangles are decision points, and triangles represent outcomes. The values for each decision point are different, as described above. Outcomes are priority decisions (defer, scheduled, out-of-cycle, immediate); outcome triangles are color coded:
 
   - Defer = gray with green outline
   - Scheduled = yellow
   - Out-of-Cycle = orange
   - Immediate = red with black outline
 
-![Suggested Supplier Tree](graphics/ssvc_2_supplier.pdf){ width=100% }
+![Suggested Supplier Tree](../graphics/ssvc_2_supplier.pdf){ width=100% }
```

Comment:
IIRC, for security reasons you can't include directory traversal in what will eventually get rendered as HTML links. So I don't think it will work for turning the markdown into a PDF. 
This isn't documented as a design requirement for the directory structure (apologies), but I believe everything needed to create a PDF should be in the `doc` directory, otherwise the pandoc scripts break. 
I can add this to `doc/style-guide-how-to` and/or the README? Unless you're able to run `doc/compile-citeproc.sh` with this reorganization and get a valid HTML file with no errors?